samples/python/hello_query_device/hello_query_device.py

```diff
@@ -27,19 +27,19 @@ def main():
     for device in core.available_devices:
         log.info(f'{device} :')
         log.info('\tSUPPORTED_METRICS:')
-        for metric in core.get_metric(device, 'SUPPORTED_METRICS'):
+        for metric in core.get_property(device, 'SUPPORTED_METRICS'):
```

Comment:
1. metric => property?
2. will you create variable for `SUPPORTED_METRICS` like it's done in C++?

Reply:
1. good catch, done
2. discussed offline